src/main/windows.ts

```diff
@@ -54,6 +54,19 @@ export function createMainWindow(): Electron.BrowserWindow {
     browserWindow = null;
   });
 
+  // Focusing on new window loads the possibly clicked "Show Me" fiddle
+  browserWindow.on('focus', () => {
+    browserWindow?.webContents.send(IpcEvents.MENU_IPC_RENDER);
+  });
+
+  // Show menu for respective window taking into consideration an entry in
+  // "Show Me" may be selected
+  ipcMain.on(IpcEvents.MENU_IPC_MAIN, (_, result) => {
```

Comment:
This IPC event will duplicate on each window creation. Should probably go elsewhere.